simperby/src/node/storage.rs

```diff
@@ -0,0 +1,176 @@
+use crate::*;
+use thiserror::Error;
+
+#[derive(Debug, Error)]
+pub enum HistoryStorageError {
+    #[error("invalid operation: {0}")]
+    InvalidOperation(String),
+    #[error("storage error: {0}")]
+    StorageError(simperby_kv_storage::Error),
+    /// When an essential item is missing or corrupted.
+    #[error("malformed storage: {0}")]
+    StorageIntegrityError(String),
+}
+
+type Error = HistoryStorageError;
+
+impl From<simperby_kv_storage::Error> for Error {
+    fn from(e: simperby_kv_storage::Error) -> Self {
+        Error::StorageError(e)
+    }
+}
+
+impl From<Error> for SimperbyError {
+    fn from(e: Error) -> Self {
+        match e {
+            HistoryStorageError::InvalidOperation(e) => SimperbyError::InvalidOperation(e),
+            HistoryStorageError::StorageError(e) => SimperbyError::StorageError(e),
+            HistoryStorageError::StorageIntegrityError(e) => {
+                SimperbyError::StorageIntegrityError(e)
+            }
+        }
+    }
+}
+
+fn create_block_key(height: u64) -> Hash256 {
+    Hash256::hash(format!("block-{}", height).as_bytes())
+}
+
+fn create_last_finlized_height_key() -> Hash256 {
+    Hash256::hash("last-finalized-height".as_bytes())
+}
+
+fn create_last_finalized_block_proof_key() -> Hash256 {
+    Hash256::hash("last-finalized-block-finalization-proof".as_bytes())
+}
+
+fn initialization_flag_key() -> Hash256 {
+    Hash256::hash("initialization_flag".as_bytes())
```

Comment:
```suggestion
    Hash256::hash("initialization-flag".as_bytes())
```
How about - instead of _ for the sense of unity?

Reply:
fixed.